src/v/redpanda/admin_server.cc

```diff
@@ -880,6 +881,11 @@ ss::future<> admin_server::throw_on_error(
             throw ss::httpd::base_exception(
               fmt::format("Too many requests: {}", ec.message()),
               ss::http::reply::status_type::too_many_requests);
+        case cluster::errc::maintenance_mode_disabled:
+            throw ss::httpd::server_error_exception(fmt::format(
+              "Maintenance mode is already disabled for node {}. Check the "
+              "status with 'rpk cluster maintenance status'.",
```

Comment:
you might remove `"status with 'rpk cluster maintenance status'.` since this api might be consumed by tools or in contexts other than rpk, so the message might not make sense in some cases.